src/remotestorage.js

```diff
@@ -181,16 +181,15 @@ RemoteStorage.prototype = {
   },
 
   /**
-   * TODO: where is this defined?
+   * @property {object} remote
```

Comment:
The `remote` property is still also documented on line 800 in that file:

```
Access to the remote backend used. Usually a <RemoteStorage.WireClient>.
```

Reply:
I removed the duplicate.